### PR TITLE
add account detail to new policy form

### PR DESCRIPTION
**Pull request by hobbitronics**

### CODE REVIEW

File: src/data/policies.ts [hunk 1]
```diff
@@ -96,6 +96,7 @@ export async function createPolicy(policyFormData: any): Promise<Policy> {
     cost_center: policyFormData.costCenter,
     entity_code: policyFormData.entityCode,
     name: policyFormData.groupName,
+    account_detail: policyFormData.accountDetail,
```
Comment (forevermatt) [inline src/data/policies.ts:99]: Would you mind alphabetizing this?

File: src/pages/policies/new.svelte [hunk 2]
```diff
@@ -69,5 +71,10 @@ const validateForm = (formData: any) => {
     <TextField placeholder="12345" bind:value={account} />
   </p>
 
+  <p>
+    <span class="header">Account Detail</span>
+    <TextField placeholder="details" bind:value={accountDetail} />
```
Comment (forevermatt) [inline src/pages/policies/new.svelte:76]: Would you mind moving the `<Description>Appears in your statements</Description>` line to here (and removing it from the Name field)? Currently, this is the field that appears in their statements, though that may change later.

Comment (hobbitronics) [reply src/pages/policies/new.svelte:75]: yup

File: src/data/policies.ts (hunk 1 above)
Comment (hobbitronics) [reply src/data/policies.ts]: no problemo

**Review by forevermatt — APPROVED** [review]

**Pull request merged**